### PR TITLE
Add: Back-to-top link for default layout

### PR DESCRIPTION
**Pull request by andygout**
cc @ironsidevsquincy 

back-to-top

### CODE REVIEW

File: client/components/back-to-top/_back-to-top.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+		border-bottom-color: transparent;
+
+		&:before {
+			@include nextIcon(arrow-up, getColor('teal-1'), 10);
+			content: '';
+			margin-left: -5px;
+			padding-right: 10px;
+		}
+
+		&:hover::before {
```
Comment (ironsidevsquincy) [inline client/components/back-to-top/_back-to-top.scss]: single colon

File: client/components/back-to-top/_back-to-top.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+			@include nextIcon(arrow-up, getColor('cold-1'), 10);
```
Comment (ironsidevsquincy) [inline client/components/back-to-top/_back-to-top.scss]: `nextIcon` has a fourth argument, `$icon-only`, which will just output the icon and exclude the width and height (as you've already set them above)

https://github.com/Financial-Times/next-sass-setup/blob/64a9065f518bc726ead3c46b17fbcb4b59a1d687/_icons.scss#L1

File: client/components/back-to-top/_back-to-top.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
+.back-to-top {
+	padding: 15px 10px 10px;
+	text-align: center;
+
+	.back-to-top__link {
+		display: block;
```
Comment (ironsidevsquincy) [inline client/components/back-to-top/_back-to-top.scss]: wonder if we want to make this full width? might it interfere with people scrolling down the screen?

Comment (andygout) [inline client/components/back-to-top/_back-to-top.scss]: Given it's at the bottom of the page does not feel so much of a problem (as opposed to having this link dotted throughout the page), and devices discriminate between the scroll/slide action of a finger and a press action, so on that basis feel keeping the touch target area big is the better option.

File: client/components/back-to-top/_back-to-top.scss (hunk 1 above)
Comment (andygout) [inline client/components/back-to-top/_back-to-top.scss]: Is it okay to use CSS3 syntax if I don't display it on IE8? https://css-tricks.com/almanac/selectors/a/after-and-before/

Comment (ironsidevsquincy) [inline client/components/back-to-top/_back-to-top.scss]: it's a touchy subject, but I'd stick to single quotes for everything

File: views/partials/back-to-top.html [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<div class='back-to-top'>
+	<a href='#' class='back-to-top__link' data-trackable='back-to-top'>Back to the top</a>
```
Comment (ironsidevsquincy) [inline views/partials/back-to-top.html]: don't know the general policy on using just `#`, but feels a bit wrong. Better link actually to something. The skip links at the top use `#primary-nav`, could use that (though looks like that's not actually on the page, so currently broken)

Comment (andygout) [inline views/partials/back-to-top.html]: Will fix `#primary-nav` in `o-header` and then add that as the 'back-to-top' link.

**Comment by ironsidevsquincy** [comment]
👍

**Pull request merged**